app/services/rss_reader.rb

```diff
@@ -40,7 +40,7 @@ def create_articles_for_user(user)
       articles.append(article)
     rescue StandardError => e
       log_error(
-        "RssReaderError: occurred while creating article",
+        "RssReaderError: occurred while creating article #{item.url}",
```

Comment:
This is left over from my debugging, I can remove it later if you want.

Reply:
All this logging will be getting an overhaul soon, you can leave it